backstack/build.gradle.kts

```diff
@@ -14,6 +14,18 @@ kotlin {
   jvm()
   ios()
   iosSimulatorArm64()
+  js(IR) {
```

Comment:
IR is the default. Can be removed from all declarations.

Reply:
nice [097ace3](https://github.com/slackhq/circuit/pull/801/commits/097ace3135a6659f33f8e9f3a5ccd8aeee5f84a3)